app/controllers/oidc_provider/sessions_controller.rb

```diff
@@ -4,7 +4,11 @@ class SessionsController < ApplicationController
 
     def destroy
       unauthenticate!
-      redirect_to OIDCProvider.after_sign_out_path
+      if OIDCProvider.after_sign_out_method
+        send(OIDCProvider.after_sign_out_method)
+      else
+        redirect_to "/"
+      end
```

Comment:
How about doing here
```suggestion
      redirect_to after_oidc_sign_out_path
```

Which is similar to what devise does with their `after_sign_out_path_for`. Then in a helper file they give a default implementation of the method and you can override it from your controller